src/state_machine/signer/mod.rs

```diff
@@ -615,8 +620,11 @@ impl<SignerType: SignerTrait> Signer<SignerType> {
                 };
 
                 info!(
-                    "Signer {} sending SignatureShareResponse for DKG round {} sign round {} sign iteration {}",
-                    signer_id, sign_request.dkg_id, sign_request.sign_id, sign_request.sign_iter_id,
+                    %signer_id,
+                    %sign_request.dkg_id,
+                    %sign_request.sign_id,
+                    %sign_request.sign_iter_id,
+                    "sending SignatureShareResponse"
```

Comment:
This call similarly adds `sign_request.` to the log line, where there is no ambiguity.